site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,276 @@
+---
+linktitle: "Binaries"
+weight: 2
+---
+
+# Creating an M3 Cluster with Binaries
+
+This guide shows you the steps involved in creating an M3 cluster using M3 binaries, typically you would automate this with infrastructure as code tools such as Terraform or [Kubernetes](/docs/operator).
+
+{{% notice note %}}
+This guide assumes you have read the [quickstart](/docs/quickstart), and builds upon the concepts in that guide.
```

Comment:
This page goes to a blank non-leaf node page:
https://deploy-preview-2795--m3db.netlify.app/docs/quickstart/

Reply:
Ah yes, I really need to get that fixed up asap so we can actually have proper sections. very next task.